starwars/starwars_people/tasks.py

```diff
@@ -0,0 +1,7 @@
+from celery import shared_task
+from starwars_people.services import CSVDataWriterAndDBSaver
+
+
+@shared_task()
+def download_dataset():
+    CSVDataWriterAndDBSaver().write_data_and_save_to_file()
```

Comment:
you need separate function where you use your api client to retrieve data from starwars api, use your adapter to adapt this data, and use CSVDataWriter to save to file etc, and call this function here